lms/services/blackboard_api/client.py

```diff
@@ -15,10 +15,17 @@
 class BlackboardAPIClient:
     """A high-level Blackboard API client."""
 
-    api = None
-
     def __init__(self, basic_client):
-        self.api = basic_client
+        self._api = basic_client
+
+    def get_token(self, authorization_code):
+        """
+        Save a new Blackboard access token for the current user to the DB.
+
+        :raise services.HTTPError: if something goes wrong with the access
+            token request to Blackboard
+        """
+        self._api.get_token(authorization_code)
```

Comment:
So view code can just do `blackboard_api_client.get_token()` instead of `blackboard_api_client.api.get_token()`. This also makes mocking simpler as the mock `BlackboardAPIClient` class doesn't need to have an `api` property that's an autospec of `BasicClient`.

Note that I _do_ think we want the actual `get_token()` implementation to live in `BasicClient`. I think that's going to be needed by future PRs that add refresh token support.